appveyor-coverage.yml

```diff
@@ -4,6 +4,7 @@ image: Visual Studio 2019
 environment:
   MIK_TEX_INSTALL_DIR: 'C:\MikTex\'
   app_version: '9.0'
+  APPVEYOR_BLOCK_DOTNETCORE_TESTS_AUTORUN: true

```

Comment:
what is this?

Reply:
Looking et the log in appveyor i saw the tthe tests were run twice. Once with nunit z and once with dotnet core.
This is a way to remove the behaviour